dxr/static/templates/text_file.html

```diff
@@ -17,7 +17,7 @@
           <ul>
             {% for item in section['items'] %}
               <li>
-                <a href="{{ item.href }}" title="{{ item.title }}" class="{{ item.icon }} icon">{{ item.title }}</a>
+                <a href="{{ item.href }}" title="{{ item.title }}" class="{{ item.icon }} icon {% if item.sync %} sync-link {% endif %}">{{ item.title }}</a>
```

Comment:
There are some extra spaces here that will result in needless spaces in the final output.